tests/classification/test_precision_recall_curve.py

```diff
@@ -119,3 +120,30 @@ def test_pr_curve(pred, target, expected_p, expected_r, expected_t):
     assert torch.allclose(p, tensor(expected_p).to(p))
     assert torch.allclose(r, tensor(expected_r).to(r))
     assert torch.allclose(t, tensor(expected_t).to(t))
+
+
+@pytest.mark.parametrize(
+    ["sample_weight", "pos_label", "exp_shape"],
+    [
+        pytest.param(1, 1.0, 42),
```

Comment:
```suggestion
        (1, 1.0, 42),
```
If there is no pytest argument, let's use simple tuple